backend/web/index.php

```diff
@@ -1,7 +1,4 @@
 <?php
-defined('YII_DEBUG') or define('YII_DEBUG', true);
-defined('YII_ENV') or define('YII_ENV', 'dev');
```

Comment:
Yeah.. that's what I get for messing with this before I am awake :/ I changed the method to include `common/config/environments.php` (handled via environments on init).